content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
+
+{{< img src="tagging/assigning_tags/hostuitags.png" alt="Infrastructure List Tags" responsive="true" style="width:80%;">}}
+
+[62]: /graphing/infrastructure/
+
+{{% /tab %}}
+{{% tab "Monitors" %}}
+
+From the [Manage Monitors][63] page, select the checkbox next to each monitor to add tags (you can select one or multiple monitors). Click the **Edit Tags** button. Enter a tag or select one used previously. Then click *Add Tag `tag:name`* or *Apply Changes*. If tags were added previously, you can assign multiples at once using the tag checkboxes.
+
+{{< img src="tagging/assigning_tags/monitortags.png" alt="Manage Monitors Tags" responsive="true" style="width:80%;">}}
+
+When creating a monitor, assign monitor tags under step 4 *Say what's happening*:
+
+{{< img src="tagging/assigning_tags/monitorindivdualtags.png" alt="Create Monitor Tags" responsive="true" style="width:80%;">}}
+
+[63]: /monitors/manage_monitor/
+
+{{% /tab %}}
+{{% tab "Distribution Metrics" %}}
+
+Assign tag keys within [Distribution Metrics][64] (Beta) to create aggregate timeseries by applying sets of tags to a metric, for which a timeseries is created for every combination of tag values within the set.
+
+**Sets of tags are limited to groups of four**:
+
+{{< img src="tagging/assigning_tags/distributionmetricstags.png" alt="Distribution Metrics Tags" responsive="true" style="width:80%;">}}
+
+[64]: /graphing/metrics/distributions/
+
+{{% /tab %}}
+{{% tab "Integrations" %}}
+
+The [AWS][60] integration tile allows you to assign additional tags to all metrics at the account level. Use a comma separated list of tags in the form `key:value`.
+
+{{< img src="tagging/assigning_tags/integrationtags.png" alt="AWS Tags" responsive="true" style="width:80%;">}}
+
+[60]: /integrations/amazon_web_services/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### API
+
+{{< tabs >}}
+{{% tab "Assignment" %}}
+
+Tags can be assigned in various ways with the [API][65]. See the list below for links to those sections:
+
+- [Post a check run][66]
+- [Post an event][67]
+- [AWS Integration][68]
+- [Post timeseries point][69]
+- [Create][70] or [Edit][71] a monitor
+- [Add][72] or [Update][73] host tags
+- [Send traces][74]
+
+[65]: /api/
+[66]: /api/?lang=python#post-a-check-run
+[67]: /api/?lang=python#post-an-event
+[68]: /api/?lang=python#aws
+[69]: /api/?lang=python#post-timeseries-points
+[70]: /api/?lang=python#create-a-monitor
+[71]: /api/?lang=python#edit-a-monitor
+[72]: /api/?lang=python#add-tags-to-a-host
+[73]: /api/?lang=python#update-host-tags
+[74]: /api/?lang=python#send-traces
+
+{{% /tab %}}
+{{% tab "Example" %}}
+
+Tagging within Datadog is a powerful way to gather your metrics
+and makes scaling your infrastructure a breeze.
+
+For a quick example to demonstrate the power of tagging, perhaps you're
+looking for a sum of two metrics, which you might normally define as follows:
```

Comment:
Maybe swap "looking for a sum of two metrics, which you might normally define as follows" with "looking for a sum of the following metrics:" to make it easier to read that the metrics are being defined in the following paragraph, not the sum.